metricbeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -5,9 +5,14 @@ This file is generated! See scripts/docs_collector.py
 [[metricbeat-module-elasticsearch]]
 == Elasticsearch module
 
-The Elasticsearch module contains a minimal set of metrics to enable monitoring of Elasticsearch across multiple versions. To monitor more Elasticsearch metrics, use our {stack-ov}/xpack-monitoring.html[monitoring] feature.
-
-The default metricsets are `node` and `node_stats`.
+The Elasticsearch module contains a minimal set of metrics to enable monitoring
+of Elasticsearch across multiple versions. The Elasticsearch X-Pack module
+enables you to monitor more Elasticsearch metrics with our {stack-ov}/xpack-monitoring.html[monitoring] feature.
+
+The default metricsets in the Elasticsearch module are `node` and `node_stats`.
+The default metricsets in the Elasticsearch X-Pack module are `ccr`,
+`cluster_stats`, `index`, `index_recovery`, `index_summary`, `ml_job`,
+`node_stats`, and `shard`.
```

Comment:
Right now, we mention the two modules, Elasticsearch and Elasticsearch X-Pack, together first. And then we mention the default metricsets for each of those two modules together in the following paragraph.

An alternative might be to cover the Elasticsearch module and its default metricsets first and cover the Elasticsearch X-Pack module and its default metricsets in the following paragraph.

I'm not sure if one is better than the other but just wanted to bring it up if you didn't consider it already. This is 100% a tech writing concerns so I'll totally defer to your judgement on it!

Reply:
I like the organization that you're suggesting because I think it would make the content easier to scan. I would suggest adding a brief introductory sentence before launching into a description of each module, though. Totally up to Lisa....

Reply:
I've reorganized the descriptions as bulleted list.  If that looks good to you, can you help re-generate the files @dedemorton ?